admin_guide/manage_nodes.adoc

```diff
@@ -267,6 +267,117 @@ Alternatively, instead of specifying specific node names (e.g., `<node1>
 <node2>`), you can use the `--selector=<node_selector>` option to evacuate pods
 on selected nodes.
 
+[[rebooting-nodes]]
+== Rebooting Nodes
+
+To reboot a node, without causing an outage for applications running on the
```

Comment:
remove comma after node (I think the clause following it can all be part of the same introductory clause)